src/main/haskell/kore/test/Test/Kore/Unification/SubstitutionNormalization.hs

```diff
@@ -119,21 +119,65 @@ test_substitutionNormalization =
       in
         testCase "Simplest cycle"
             (assertEqual ""
-                (Left (CtorCircularVariableDependency [var1]))
+                (Right
+                    [    ( asVariable (v1 PatternSort)
```

Comment:
If you really feel like it, you could make sure that normalization removes x=x assignments, but leaving a TODO in the normalization code is also fine.